lib/aranea/failure_repository.rb

```diff
@@ -45,7 +49,11 @@ def respond!
       elsif @response == 'ssl_error'
         raise ::Faraday::SSLError, 'Fake failure from Aranea'
       else
-        ::Faraday::Response.new(status: @response.to_i, body: 'Fake failure from Aranea', response_headers: {})
+        ::Faraday::Response.new(
+          status: @response.to_i,
+          body: @response_hash.to_json,
+          response_headers: @response_headers_hash
+        )
```

Comment:
so it _was_ sending a response before, we just couldn't control it?

Reply:
That's right - it was sending a generic textual response.

When using the Eureka-client gem with this version, it attempts will throw an error because of:
https://github.com/mdsol/eureka-client/blob/ac3dbb941af98e3d44a3a5d9917f65b606490a2c/lib/google/api_client/result.rb#L62

This is because we don't specify the content type, and it fails to unmarshal.

Reply:
good fix!